src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -608,11 +616,14 @@ def __init__(self, module_inst, fscid):
         self.fscid = fscid
 
     @ViewCache()
-    def get(self):
+    def get(self, suppress_errors='True'):
         try:
             ret = CephService.send_command('mds', 'session ls', srv_spec='{0}:0'.format(self.fscid))
         except RuntimeError:
-            ret = []
+            if ast.literal_eval(suppress_errors):
+                ret = []
+            else:
+                raise
```

Comment:
I would suggest using a better error message for this scenario too
```suggestion
            if ast.literal_eval(suppress_errors):
                ret = []
            else:
                raise cherrypy.HTTPError(500, "No active MDS found ")
```

Reply:
500 is raised in _clients(), another advantage of it is that we have a clear concise list of exceptions that tell us what will be returned - all at one place. What you suggested makes sense but i'm more inclined towards handling all cases in the helper _clients() **OR** all the exceptions can be moved to `get()` (from `_clients()`)

Reply:
true, forgot about it. Thanks!